src/lib/utils/analytics.ts

```diff
@@ -29,12 +29,18 @@ export const getPriceUpdateBasisPoints = (
   return formatPercentInBasisPointsNumber(changePercentage)
 }
 
+function getEstimatedNetworkFee(trade: InterfaceTrade) {
+  if (isClassicTrade(trade)) return trade.gasUseEstimateUSD
```

Comment:
should `isClassicTrade` return `trade.classicGasUseEstimateUSD`?

Reply:
no, thats the classic gas amount for comparison when the actual trade is uniswapx